### PR TITLE
Meta in linkages

### PR DESCRIPTION
**Pull request by hhware**
Implements suggestion from https://github.com/json-api/json-api/pull/431#issuecomment-92938203. Please see #415 and #431 for more details.

I would like to request to consider this for 1.0.

### CODE REVIEW

**Comment by tkellen** [comment]
:+1: from me. @dgeb?

File: format/index.md [hunk 1]
```diff
@@ -309,6 +309,8 @@ Resource linkage **MUST** be represented as one of the following:
 
 A "linkage object" is an object that identifies an individual related resource.
 It **MUST** contain `type` and `id` members.
+A linkage object **MAY** include `"meta"` member to contain non-standard
```
Comment (dgeb) [inline format/index.md]: Please add an `a` after `include` and then I'm :+1:

**Comment by hhware** [comment]
Updated, thanks!

**Comment by dgeb** [comment]
Thanks @hhware!

**Pull request merged**